extensions/sql-database-projects/src/models/project.ts

```diff
@@ -70,6 +70,24 @@ export class Project {
 					this.files.push(this.createProjectEntry(folderElements[f].getAttribute(constants.Include), EntryType.Folder));
 				}
 			}
+
+			// find all pre-deployment scripts to include
+			const preDeploy = itemGroup.getElementsByTagName(constants.PreDeploy);
+			for (let p = 0; p < preDeploy.length; p++) {
+				this.files.push(this.createProjectEntry(preDeploy[p].getAttribute(constants.Include), EntryType.File));
```

Comment:
How does adding these to `files` affect Build?  My gut says they should each be a separate property in `Project` since they're special (and to enforce that there's only one).  Or at least a new `EntryType` value.

Reply:
As discussed offline, adding the scripts to "files" wouldn't impact Build since Build takes sqlproj into consideration. 
But I have updated these scripts to have their own separate property and updated the printing of tree.